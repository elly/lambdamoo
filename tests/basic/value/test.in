typeof(3) == INT
typeof("abc") == STR
typeof(1.0) == FLOAT
typeof({"foo"}) == LIST
typeof(#0) == OBJ
tostr(17)
tostr(3.0)
tostr(#0)
tostr({1, 2})
toliteral(17)
toliteral(3.0)
toliteral(#0)
toliteral({1, 2})
toint(10)
toint(3.0)
toint(#2)
toint("-34")
toobj("2")
toobj(2)
toobj("foo")
tofloat(34)
tofloat("34.7")
tofloat(7.3)
equal(1 + 1, 2)
equal("Foo", "foo")
equal("foo", "foo")
equal({1, 2}, {1, 2})
equal({1, 2}, {1, 3})
