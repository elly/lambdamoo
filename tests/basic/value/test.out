1
1
1
1
1
"17"
"3.0"
"#0"
"{list}"
"17"
"3.0"
"#0"
"{1, 2}"
10
3
2
-34
#2
#2
#0
34.0
34.7
7.3
1
0
1
1
0
